apm-dist/src/main/assembly/binary.xml

```diff
@@ -69,6 +69,7 @@
                 <include>ui-initialized-templates/*/*.json</include>
                 <include>lal/*</include>
                 <include>log-mal-rules/*</include>
+                <include>telegraf-rules/*.yaml</include>

```

Comment:
```suggestion
                <include>telegraf-rules/*</include>

```
Why do we need a `yaml` suffix? I assume there is no other files in the folder, right?